### PR TITLE
Congress state endpoint

### PR DESCRIPTION
**Pull request by itemniner**
## Type of change made:
- [x] **Bugfix** (non-breaking change which fixes an issue)
- [x] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected)
- [x] **Refactoring** (improves existing functionality)
- [x] **Testing** (tests existing code or improves existing tests)
- [ ] **Styling** (no functionality changed, just visual improvements)
## Description:
This code hides the environmental variables for the propublica api . Also setup testing for the endpoints. This code contains the api call which is parsed and made into a POPO and displayed as JSON formatted response when the endpoint is requested.
## What functionality does this accomplish?
- returns parsed data for Representatives of senates index page
closes #
## Checklist:
- [ ] My code has no unused/commented out code
- [x] I have reviewed my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have fully tested my code
- [ ] I have partially tested my code (please explain why):

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -56,6 +63,26 @@ def get_by_id(id_):
...
 
+@app.route("/members_by_state/<state_>")
+def get_members_by_state(state_):
+    headers = {'X-API-Key': os.getenv('PROP_API')}
+    URL = f'https://api.propublica.org/congress/v1/members/senate/{state_}/current.json'
+    response = requests.get(URL, headers = headers).json()
+    results = response['results']
+
+    objects = map(lambda result: MemberIndex(first_name = result['first_name'],
+                                             party = result['party'],
```
Comment (itemniner) [inline app.py:74]: Possibly add house reps to the same method to get them all in one endpoint. We should look into services but the way modules are called still confuse me. I think all you need is "import"? I wonder how it works when its nested in folders. Something to research...

File: Pipfile [hunk 2]
```diff
@@ -37,6 +37,8 @@ flask-cors = "*"
 pytest-cov = "*"
 pytest-flask = "*"
 requests = "*"
+marshmallow = "*"
+python-dotenv = "*"
```
Comment (SMJ289) [inline Pipfile:42]: We may not need the marshmallow module so we can remove it later.

File: app.py [hunk 3]
```diff
@@ -56,6 +63,26 @@ def get_by_id(id_):
...
+                                             last_name =result['last_name']),
+                                             results)
+    list_of_members = list(objects)
+
+    try:
+        return jsonify({"results": list(map(lambda member: member.serialize(), list_of_members))})
+
+    except Exception as e:
+	    return(str(e))
+
```
Comment (SMJ289) [inline app.py:85]: We need to figure out how to all the senators and house reps together in one result.

File: app.py (hunk 1 above)
Comment (jklich151) [reply app.py:74]: I agree, it'd be nice to get them all in one call.

**Pull request merged**